src/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ find_package(PkgConfig REQUIRED)
 find_package(Qt5 COMPONENTS Network REQUIRED)  # probono: For qtsingleapplication
 find_package(dbusmenu-qt5 REQUIRED)
 find_package(KF5GlobalAccel REQUIRED)
-pkg_check_modules(QCALCULATE REQUIRED libqalculate)
+pkg_check_modules(QALCULATE REQUIRED libqalculate)
```

Comment:
this of course isn't required, but it seemed reasonable to use the same spelling as the module